docs/getting-started.md

```diff
@@ -30,16 +30,16 @@ you'll need to acquire them on your own.
     MD5 (mcpx_1.0.bin) = d49c52a4102f6df7bcf8d0617ac475ed
 
 If your MCPX dump has an MD5 of `196a5f59a13382c185636e691d6c323d`, you dumped
-it badly and it's a couple of bytes off. It should start with 0x33 0xC0 and end
-with 0x02 0xEE.
+it badly and it's a couple of bytes off. It should start with `0x33 0xC0` and end
+with `0x02 0xEE`.
 
 #### Flash ROM Image (aka BIOS)
```

Comment:
Shall we also add " / Kernel" here?

Reply:
Hmm.. well they aren't the same thing, but the layperson may refer to them this way so I'll add it